frontend/packages/db/prisma/migrations/20250311080742_add_repository_model/migration.sql

```diff
@@ -0,0 +1,40 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `installationId` on the `PullRequest` table. All the data in the column will be lost.
+  - You are about to drop the column `repositoryName` on the `PullRequest` table. All the data in the column will be lost.
+  - You are about to drop the column `repositoryOwner` on the `PullRequest` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[repositoryId,pullNumber]` on the table `PullRequest` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `repositoryId` to the `PullRequest` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- DropIndex
+DROP INDEX "PullRequest_repositoryOwner_repositoryName_pullNumber_key";
+
+-- AlterTable
+ALTER TABLE "PullRequest" DROP COLUMN "installationId",
+DROP COLUMN "repositoryName",
+DROP COLUMN "repositoryOwner",
+ADD COLUMN     "repositoryId" INTEGER NOT NULL;
+
+-- CreateTable
+CREATE TABLE "Repository" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "owner" TEXT NOT NULL,
+    "installationId" BIGINT NOT NULL,
+    "isActive" BOOLEAN NOT NULL DEFAULT true,
```

Comment:
There may be options like:

- Making it nullable and changing it to a date type.
- Creating a separate table and changing it to a date type (like for deactivation).

However, I think this simple approach is still the best. :+1: